src/app/components/canvas/canvas.component.ts

```diff
@@ -427,6 +427,11 @@ export class CanvasComponent implements OnInit, OnDestroy {
     const title = this.board.task.title ? this.board.task.title : 'No task created!';
     const message = this.board.task.message;
 
+    if (this.snackbarService.snackBar._openedSnackBarRef) {
```

Comment:
Could we make a function inside snackbarService to see if there's a currently-opened snack bar instead? 

We should keep the 'snackBar' field private and inaccessible from other classes. I forgot to do this initially.

Reply:
makes sense, just added this change